docs/tasks/administer_cluster_resources.md

```diff
@@ -0,0 +1,100 @@
+# Administer Cluster resources
+
+This page shows you how to represent your cluster resource limits and to
+establish fair sharing rules among the tenants.
+
+The intended audience for this page are cluster administrators.
+
+## Before you begin
+
+You need to have a Kubernetes cluster, the kubectl command-line tool
+must be configured to communicate with your cluster, and [Kueue installed](/README.md#installation).
+
+## Single Cluster Queue setup
+
+### 1. Create a [Cluster Queue](/docs/concepts/cluster_queue.md)
+
+Create a single ClusterQueue to represent the resource limits for your
+entire cluster.
+
+```shell
+kubectl apply -f cluster-total.yaml
+```
+
+```yaml
+# cluster-total.yaml
+apiVersion: kueue.x-k8s.io/v1alpha1
+kind: ClusterQueue
+metadata:
+  name: cluster-total
+spec:
+  namespaceSelector: {} # match all.
+  requestableResources:
+  - name: "cpu"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 9
+        ceiling: 9
+  - name: "memory"
+    flavors:
+    - resourceFlavor: default
+      quota:
+        guaranteed: 36Gi
+        ceiling: 36Gi
+```
+
+This ClusterQueue governs the usage of [resource types](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-types)
+`cpu` and `memory`. Each resource type has a single [resource flavor](/docs/concepts/cluster_queue.md#resource-flavors),
+named `default` with a guaranteed quota.
+As there is a single ClusterQueue in the cluster, it is not possible for the
+ClusterQueue to borrow resources from anywhere. Thus, the `ceiling` is equal
```

Comment:
btw, in the docs we mention that [no ceiling means unlimited](https://github.com/kubernetes-sigs/kueue/blob/27b05145f94ac528ff92790efc70d62d8c9feafc/api/v1alpha1/clusterqueue_types.go#L250), which is currently not true, the ceiling must be explicitly set to guaranteed. I opened https://github.com/kubernetes-sigs/kueue/issues/144

Reply:
Sg. I will remove the ceiling line once when I fix #144.